build.xml

```diff
@@ -334,7 +334,7 @@
       <javadoc destdir="@{destdir}" author="true" version="true" use="true"
         windowtitle="${ant.project.name} API" classpathref="cassandra.classpath"
         bottom="Copyright &amp;copy; 2009- The Apache Software Foundation"
-        useexternalfile="yes" encoding="UTF-8" failonerror="false"
+        useexternalfile="yes" encoding="UTF-8" failonerror="true"
```

Comment:
After reviewing the whole patch I am inclined to suggest only big fat warning on broken task and not to fail builds.
I can imagine how that will slow down big CEPs as people will see it most probably in CI when they run check. I would prefer follow up tickets. IDE does not warn you on all potential errors, not sure if we can change that by default for the project? With the current setup this will slow down and annoy people who do big features development.